src/mvAppItem.cpp

```diff
@@ -4086,13 +4205,30 @@ DearPyGui::GetEntityParser(mvAppItemType type)
             MV_PARSER_ARG_SHOW)
         );
 
-        args.push_back({ mvPyDataType::Double, "x" });
-        args.push_back({ mvPyDataType::Double, "y" });
-        args.push_back({ mvPyDataType::Integer, "x_offset", mvArgType::KEYWORD_ARG });
-        args.push_back({ mvPyDataType::Integer, "y_offset", mvArgType::KEYWORD_ARG });
+        args.push_back({ mvPyDataType::DoubleList, "x" });
```

Comment:
I think it was because of consistency with all the other series... I'm pretty sure I tried to accept single elements but I couldn't. Anyway I'll try again and I'll tell you if and where I get stuck

Reply:
I made it, but now it accepts only double (not int) parameters.. I don't actually know why it doesn't cast them automatically

Reply:
> but now it accepts only double (not int) parameters

This is because both `isPyObject_Double` (called from within the parser) and `ToDouble`, unlike their Float counterparts, assume that the Python value may only be a floating-point number. Fixing that seems to be beyond the scope of this update; I'd prefer to do it in a separate PR.

However, there's still one problem with the current implementation: if `x`/`y` are of unsupported type (other than floating-point number, e.g. int), the app will _crash_ in `draw_label_series` because it unconditionally takes the 0-th element of x/y arrays, which may well be empty. Let's add a size check to `draw_label_series` in order to prevent the crash.

Reply:
And in that case what should I do? Putting a default value like 0 or skipping the draw part (I'm pretty sure it'd break other things)?

Reply:
Skipping the draw should be safe if we quit the draw function early (just like many of the draw functions check `config.show` and quit if it's false).